modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -60,23 +60,26 @@ Suppose we have the next configuration:
+If the processors within the `try` Scope do not produce any error, the scope finishes the execution and commits the transaction, independently of the configured `transactionalAction` value.
 
-We will analyze what happens with the transaction and messages with the `<raise-error/>` processor added, depending on the `transactionalAction` of the `try` scope:
+The transaction and the messages are handled differently when a `<raise-error/>` processor is added, depending on the `transactionalAction` of the `try` scope:
 
-* Ignore (`INDIFFERENT`): in this case, the transaction is _suspended_ while executing the Try Scope inner processors. When the error is raised, it is propagated to the source (which does not handle it with an `<on-error-continue>`). So, the transaction is rolled back, and the message will be available again in the JMS queue. However, the `jms:publish` operation was completed and the message has been sent, since the rollback of the transaction does not affect the publish operation, which was outside of the transaction scope (even though the `transactionalAction` of the `jms:publish` operation is `BEGIN_OR_JOIN`). This will behave the same as if, instead of having a Try Scope with a `transactionalAction="INDIFFERENT"`, we had the `jms:publish` with `transactionalAction="NOT_SUPPORTED"`.
-* Always Begin (`ALWAYS_BEGIN`): in this case a new transaction is initiated for the execution of the try scope. After the error is raised (and not handled), the new transaction is rolled back, which means that the `jms:publish` operation is not committed and thus the message is not actually sent to the `test.out` queue. Then, the transaction of the flow (which was began by the message source) is also rolled back, and the message is available again in the `test.in` queue.
-* Begin or Join (`BEGIN_OR_JOIN`): in this case, when the error is raised, it is not inmediately rolled back. Since the flow might be able to handle the raised error and commit the transaction, the `jms:publish` operation can still complete and send the message. This time, since the error is not handled with an `on-error-continue`, then the entire transaction is rolled back and the result is the same as in the case of Always Begin.
+* Ignore (`INDIFFERENT`): In this case, the transaction is continued while executing the processors inside the Try scope. When the error is raised, it is propagated to the source (which does not handle it with an `<on-error-continue>`). The transaction is rolled back, and the message will be available again in the JMS queue. However, the `jms:publish` operation was completed and the message has been sent since the rollback of the transaction does not affect the publish operation, which was outside of the transaction scope (the `transactionAction` is set to its default, `NOT_SUPPORTED`).
```

Comment:
COPYEDITS:
"In this case, the transaction is continued..."
  >  "Tthe transaction continues...."

"...with an `<on-error-continue>`)."
  >  "...with `<on-error-continue>`)."

"...back, and the message will be available again in the JMS queue."
  >  "...back, and the message is available again in the JMS queue."

"However, the `jms:publish` operation was completed and the message has been sent since the rollback of the transaction does not affect the publish operation, which was outside of the transaction scope (the `transactionAction` is set to its default, `NOT_SUPPORTED`)."
  >
"This rollback does not affect the completed `jms:publish` operation, which was outside of the transaction scope because `transactionAction` was set to its default, `NOT_SUPPORTED`). Had 'transactionAction` been set to 'JOIN_IF_POSSIBLE," the completed operation would have been rolled back.
(Did I understand this correctly?)